packages/docs/src/styleguide.md

```diff
@@ -1,16 +1,18 @@
-Blueprint is a React UI toolkit for the web.
+Blueprint is a React-based UI toolkit for the web.
```

Comment:
Looks like I'm the only one thinking like this, so nevermind.

Can you make sure you caught all instances of that tagline and that it's changed everywhere?